packages/web/src/components/spa/HackAUBG/JourneySection/JourneyDesktop/JourneyStep.jsx

```diff
@@ -0,0 +1,37 @@
+export const JourneyStep = ({ props }) => {
```

Comment:
So, if you are to pass props as such, it should be `(props) => ` instead of defining a object with the key `props`:
This will enable you to do this:
```jsx
<JourneyStep title={...} icon={...}/>
// or
<JourneyStep {...props} />
```
Nevertheless, since we're not using PropTypes or Typescript, it's better you define the props by hand so that you get autocompletion when passing props.